src/containers/scenes/data-scene/data-scene.js

```diff
@@ -76,8 +75,19 @@ const Container = (props) => {
   }
 
   const handleTooltipActionButtonClick = () => {
-    precomputedAoiAnalytics(mapTooltipContent.title);
-    browsePage({ type: AREA_OF_INTEREST, payload: { id: mapTooltipContent.id }, query: { precalculatedLayer: selectedAnalysisLayer.slug, OBJECTID: mapTooltipContent.objectId } });
+    const { title } = mapTooltipContent;
+    const { isSubnational } = mapTooltipData;
+    precomputedAoiAnalytics(title);
+
+    const isAdminNational = areaTypeSelected === AREA_TYPES.administrative && !isSubnational;
+    const isAdminSubnational = areaTypeSelected === AREA_TYPES.administrative && isSubnational;
+
+    const precalculatedLayer = () => {
```

Comment:
We have to remember to refactor this

Reply:
Added a subtask [HE-519](https://vizzuality.atlassian.net/browse/HE-519)